babel.config.js

```diff
@@ -27,6 +27,18 @@ module.exports = function (api) {
           loose: true,
         },
       ],
+      [
+        "@babel/plugin-proposal-private-methods",
+        {
+          loose: true,
+        },
+      ],
+      [
+        "@babel/plugin-proposal-private-property-in-object",
+        {
+          loose: true,
+        },
+      ],
```

Comment:
`@babel/plugin-proposal-class-properties`, `@babel/plugin-proposal-private-methods` and `@babel/plugin-proposal-private-property-in-object` all need to have the same option for `loose`, per babel docs [here](babel/plugin-proposal-private-methods#loose) and [here](https://babeljs.io/docs/en/babel-plugin-proposal-private-property-in-object#loose).